### ISSUE TITLE
test: replication/gc* tests fail on 1.10 branch after replication/catch test

### ISSUE THREAD

**Issue by avtikhon** [tarantool/tarantool#4590]
Tarantool version:
1.10 branch

OS version:
Ubuntu 18.04/Debian 9

Bug description:
```
[001] Test failed! Result content mismatch:
[001] --- replication/gc.result	Mon Oct 28 10:15:23 2019
[001] +++ replication/gc.reject	Mon Oct 28 16:22:36 2019
[001] @@ -124,7 +124,16 @@
[001]  ...
[001]  #fio.glob('./master/*.xlog') == 1 or fio.listdir('./master')
[001]  ---
[001] -- true
[001] +- - 00000000000000000113.xlog
[001] +  - 00000000000000000326.vylog
[001] +  - 00000000000000000002.xlog
[001] +  - 00000000000000000216.xlog
[001] +  - 00000000000000000426.xlog
[001] +  - 00000000000000000276.xlog
[001] +  - 00000000000000000426.snap
[001] +  - '513'
[001] +  - 00000000000000000105.xlog
[001] +  - 00000000000000000326.xlog
[001]  ...
```
Steps to reproduce:
./test-run.py replication/catch.test.lua replication/gc.test.lua

Optional (but very desirable):
* coredump
* backtrace
* netstat

**Comment by avtikhon** [comment]
Fix  backported.

**Comment by Totktonada** [comment]
Closed in 669c1a2f7821ceb4a6d2f85c3e23341298a67dc7 AFAIU.
